README.md

```diff
@@ -1,80 +1,190 @@
+The application is built using the following structure:
 
-We do this because instagram has a 5000-requests an hour per application limit for applications where the user is not authenticated.
+* client (main folder for client side only code).
+    * common (holds common code such as global helpers)
+    * less
+    * modules (each logical piece of the application is treated like a module)
+        * some module
+            * some_root_level.html or .js
+            * some submodule folder
+                * some submodule .html or .js
+* lib (main folder for code shared between client and server).
+    * It may seem a little strange that the 'lib' folder doesn't actually contain any external libraries. We have named this folder lib as per Meteor's own [recommendations](http://docs.meteor.com/#/full/structuringyourapp) for structuring an app.
+* public (assets).
+* server (main folder for server side only code).
+    * modules (some of the client side modules also have server side counter parts. Others exist only on the server).
+        * some submodule
+            * some submodule .js
+#### 2. Routing
+Meteor applications mostly use one package for routing: the [iron:router](https://atmospherejs.com/iron/router). Using a router is essential for a larger application. With Meteor, using a router changes some of the basic mechanics of the framework. For example: In the chat application, you simply published a collection and then subscribed to it from the client. With a router, this needs to be handled for each route to make sure that a subscription is ready once a route function is executed. In the next series of subtasks, you will learn the parts of iron:router that are needed for Tweeteor.
+
+##### Registering a route
+Tweeteor requires two routes. One root route that shows posts and one route for the profile page. Open up *router.js* and add these two routes. The profile route should use a parameter that is unique for each user, the username for instance.
+
+Create a data function for both routes that for now just return a predefined set of data so that you can verify that your routes work.
+
+*Tip: Iron router looks for templates named the same as you routes*
+
+*Tip: Look at the templates provided for posts and profile. Add some temporary code here to display the dummy data you put in the data function of your routes*
+
+##### Layout
+Right now, your app probably doesn't look to good. Let's fix this by using the layout we have provided. If you open *layout.html* you can see that this file has some default markup. 
+
+To use this layout in your router, open *router.js* and create a *Router.configure* function that takes in an object. Add a *layoutTemplate* property on this object with a string that matches the name of the layout template in *layout.html*. 
+
+To wrap it all up you need to tell the router where to *yield* the template rendered in the route function. Adding the special template function {{> yield}} does the trick.
```

Comment:
To wrap it all up you need to tell the router where to render the template. Using spacebars's [inclusion-tag](https://github.com/meteor/meteor/blob/devel/packages/spacebars/README.md#inclusion-and-block-arguments) (```{{> insert_template_name}}``) does the trick.